packages/react-instantsearch-core/src/connectors/connectVoiceSearch.js

```diff
@@ -0,0 +1,159 @@
+import PropTypes from 'prop-types';
+import createConnector from '../core/createConnector';
+import {
+  cleanUpValue,
+  refineValue,
+  getCurrentRefinementValue,
+  getIndexId,
+} from '../core/indexUtils';
+
+function getId() {
+  return 'query';
+}
+
+function getAdditionalId() {
```

Comment:
Why not using constants? (same for `getId`)

Reply:
consistent with the other connectors.